spec/presenters/change_history_presenter_spec.rb

```diff
@@ -1,83 +1,87 @@
-        expect(subject).to eq details[:change_history]
+    context "when the details hash does not have a change history" do
+      let(:details) { {} }
+      let(:change_notes) do
+        [
+          build(:change_note, note: "Note 1", public_timestamp: 3.days.ago),
+          build(:change_note, note: "Note 2", public_timestamp: 2.days.ago),
+          build(:change_note, note: "Note 3", public_timestamp: 1.day.ago),
+        ]
       end
-    end
 
-    context "details hash doesn't include content_history" do
+      let(:query_response) { change_notes }
+
       before do
-        2.times do |i|
-          create(:change_note, edition:, note: i.to_s, public_timestamp: Time.zone.now.utc)
-        end
+        expect(Presenters::Queries::ChangeHistory).to receive(:new)
+                                                        .with(edition, include_edition_change_history: true)
+                                                        .and_return(change_history_stub)
       end
-      it "constructs content history from change notes" do
-        expect(subject.map { |item| item[:note] }).to eq %w[0 1]
-      end
-    end
 
-    it "orders change notes by public_timestamp (ascending)" do
-      [1, 3, 2].to_a.each do |i|
-        create(:change_note, edition:, note: i.to_s, public_timestamp: i.days.ago)
+      it "returns the change notes from the database sorted by date" do
+        expect(subject).to eq([
+          { public_timestamp: change_notes[2].public_timestamp.utc.to_s, note: change_notes[2].note },
+          { public_timestamp: change_notes[1].public_timestamp.utc.to_s, note: change_notes[1].note },
+          { public_timestamp: change_notes[0].public_timestamp.utc.to_s, note: change_notes[0].note },
+        ])
       end
-      expect(subject.map { |item| item[:note] }).to eq %w[3 2 1]
     end
 
-    it "omits change notes that don't have a public timestamp" do
-      create(:change_note, edition:, note: "with-timestamp", public_timestamp: 1.day.ago)
-      create(:change_note, edition:, note: "without-timestamp", public_timestamp: nil)
-      expect(subject.map { |item| item[:note] }).to eq %w[with-timestamp]
-    end
+    context "when the details hash has a change history" do
+      let(:one_day_ago) { 1.day.ago }
+      let(:two_days_ago) { 2.days.ago }
 
-    context "multiple editions for a single content id" do
-      let(:item1) do
-        create(
-          :superseded_edition,
-          document:,
-          details:,
-          user_facing_version: 1,
-        )
-      end
-      let(:item2) do
-        create(
-          :live_edition,
-          document:,
-          details:,
-          user_facing_version: 2,
-        )
+      let(:details) do
+        { change_history: [
+          { public_timestamp: one_day_ago.in_time_zone("GMT").to_s, note: "note 1" },
+          { public_timestamp: two_days_ago.in_time_zone("GMT").to_s, note: "note 2" },
+        ] }
```

Comment:
Should these use the `utc` method, to remain consistent with the other tests (instead of `in_time_zone("GMT")`)?